whelk-core/src/main/groovy/whelk/component/ElasticSearch.groovy

```diff
@@ -185,12 +185,14 @@ class ElasticSearch {
         whelk.embellish(copy, filterOutNonChipTerms)
     }
 
-    private static void setComputedProperties(Document doc) {
+    private static void setComputedProperties(Document doc, Whelk whelk) {
         getOtherIsbns(doc.getIsbnValues())
                 .each { doc.addTypedThingIdentifier('ISBN', it) }
 
         getOtherIsbns(doc.getIsbnHiddenValues())
                 .each { doc.addIndirectTypedThingIdentifier('ISBN', it) }
+
+        doc.data['@graph'][0]['linksHereCount'] = whelk.getStorage().getDependers(doc.getShortId()).size()
```

Comment:
Looks promising! Let's just model this a bit before merging. (It could, in the output, be expressed as a link to an `?o`-search for the item, with a totalItems count.)